Source/Core/Common/CMakeLists.txt

```diff
@@ -164,3 +164,17 @@ if(UNIX)
 elseif(WIN32)
   target_link_libraries(common PRIVATE "-INCLUDE:enableCompatPatches")
 endif()
+
+find_package(Qt5 5.9 REQUIRED COMPONENTS Core Network)
```

Comment:
Sorry, but Qt shouldn't be present in anything outside of DolphinQt. No UI frameworks should be touching anything other than UI code.